resources/package.json

```diff
@@ -1,5 +1,5 @@
 {
   "main": "../build/cjs/resources.js",
   "module": "../build/esm/resources.js",
-  "types": "../build/esm/resources.d.ts"
```

Comment:
🤔 Don't think this is the problem. From what I can see there is no `resources.x` at all, in any of the build folders

Reply:
Ok, was expecting an index inside resources folder. Instead we have both a `resources.js` and a `resources` folder. It might cause some headaches in future...

Anyway seems like we export types from `cjs` folder in the root level, so let's keep it consistent